portfolio/src/main/webapp/index.html

```diff
@@ -50,7 +51,7 @@ <h1>
             </h1>
             <p>
                 Some of the projects I have worked on: <br />
-                Android app: story sticker; orchardwatch (app for local apple farm) <br />
+                Android app: story creation page in Facebook story; orchardwatch (app for local apple farm) <br />
```

Comment:
small nit: this change shouldn't technically be in this PR since it's unrelated. Fine to leave for now though!